src/calculateRentalCost.js

```diff
@@ -4,7 +4,27 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40; // dolars$
+  const longTerm = 7; // days
+  const longTermDiscount = 50; // dolars$
+  const shortTerm = 3; // days
+  const shortTermDiscount = 20; // dolars$
+  let total = days * pricePerDay;
+
+  switch (true) {
```

Comment:
no no no, we shouldn't move that way, it's magic string i can't understand why true what does it mean
let's follow checklist